tests/integration/targets/prepare_iosxr_tests/aliases

```diff
@@ -0,0 +1 @@
+network/iosxr
```

Comment:
change this to `network/yang` after adding the `yang` group in inventory template Refer
1) https://github.com/ansible/ansible-zuul-jobs/blob/1586f4e2aad2a211c7c7b196a937fd26d8c86abd/roles/ansible-test-inventory/templates/inventory-iosxr.j2#L7

2) https://github.com/ansible/ansible-zuul-jobs/blob/1586f4e2aad2a211c7c7b196a937fd26d8c86abd/roles/ansible-test-inventory/templates/inventory-iosxr.j2#L7